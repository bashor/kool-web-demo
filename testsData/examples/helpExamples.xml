<?xml version="1.0"?>
<help>
    <example>
        <name>Simplest version</name>
        <text>We declare a package-level function main which returns Unit and takes an Array of strings as a
            parameter.
            <br/>
            Note that semicolons are optional.
        </text>
        <args/>
	 <target>java js</target>
    </example>
    <example>
        <name>Reading many names from the command line</name>
        <text>Line 2 demonstrates the
            <a href="http://confluence.jetbrains.net/display/Kotlin/Control+structures#Controlstructures-Forloop">
                <b>for</b>-loop</a>
            , that would have been called "enhanced" if there were any other for-loop in Kotlin.
        </text>
        <args>guest1 guest2 guest3</args>
        <target>java js</target>
    </example>
    <example>
        <name>An object-oriented Hello</name>
        <text>Here we have a
            <a href="http://confluence.jetbrains.net/display/Kotlin/Classes+and+Inheritance">class</a>
            with a
            <i>primary constructor</i>
            and a member function.
            <br/>
            Note that there's no
            <b>new</b>
            keyword used to create an object.
        </text>
        <args>guest1</args>
        <target>java js</target>
    </example>
</help>
