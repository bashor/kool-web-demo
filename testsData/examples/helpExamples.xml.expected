[{target=java js, text=We declare a package-level function main which returns Unit and takes an Array of strings as a
            parameter.
            <br></br>
            Note that semicolons are optional.
        , args=, name=Simplest version}, {target=java js, text=Line 2 demonstrates the
            <a target="_blank" href="http://confluence.jetbrains.net/display/Kotlin/Control+structures#Controlstructures-Forloop">
                <b>for</b>-loop</a>
            , that would have been called "enhanced" if there were any other for-loop in Kotlin.
        , args=guest1 guest2 guest3, name=Reading many names from the command line}, {target=java js, text=Here we have a
            <a target="_blank" href="http://confluence.jetbrains.net/display/Kotlin/Classes+and+Inheritance">class</a>
            with a
            <i>primary constructor</i>
            and a member function.
            <br></br>
            Note that there's no
            <b>new</b>
            keyword used to create an object.
        , args=guest1, name=An object-oriented Hello}]